pkgs/build-support/rust/fetchcrate.nix

```diff
@@ -17,4 +17,4 @@ assert pname == null || pname == crateName;
   url = "${registryDl}/${crateName}/${version}/download";
 } // lib.optionalAttrs unpack {
   extension = "tar.gz";
-} // removeAttrs args [ "crateName" "pname" "version" "unpack" ])
+} // removeAttrs args [ "crateName" "pname" "registryDl" "version" "unpack" ])
```

Comment:
I assume you wanted to add it alphabetically, but keep the last one unordered?

Reply:
I just followed the order of the inputs, tbh I don't really remember why I ordered inputs like that